core/utils.lua

```diff
@@ -584,3 +584,43 @@ function format_ui_value(value)
 end
 
 --#endregion
+
+
+function poll_seal(_key, _mod, _guaranteed, _options, _type_key)
```

Comment:
Consistency, this should be `SMODS.poll_seal`. API design, this has too many arguments. I suggest converting to a table argument that can hold these arguments as needed